assemble-lids.service.in

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Assemble code update image
+
+[Service]
+Type=oneshot
```

Comment:
what does this type mean?

Reply:
This seems to be a good example: https://trstringer.com/simple-vs-oneshot-systemd-service/

The oneshot argument makes the service wait until the ExectStart commands are done, otherwise the systemd will immediately issue a success signal. In this case we want systemd to wait until the gen-lids-image-tar is complete in case there's an error, so that systemd can emit a failure to start the service file.